bundle/2.4.0/cockroachdb-certified-rhmp/manifests/cockroach-operator-webhook-service_v1_service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
```

Comment:
We used to "combine" these all into one yaml file - probably doesn't matter if we don't?

Reply:
Correct. This isn't needed so I opted for simple